index.css

```diff
@@ -1,246 +1,130 @@
-:root {
-  --primary-color: #6070FF;
-  --secondary-color: #7F8CFF;
-  --title-color: #172B4D;
-  --text-color: #344563;
-  --label-background-color: #EBEBFF;
-}
-
 body {
-  background-color: #F7F7F9;
+  background-color: #f7f7f9;
 }
-html *{
-  font-family: 'Poppins';
+
+html * {
+  font-family: 'Poppins', Arial, Helvetica, sans-serif;
+  margin: 0;
 }
 
 @media (min-width: 375px) {
-  #navbar {
-   display: flex;
-   justify-content: space-between;
-   height: 48px;
-   width: 375px;
-   top: 44px;
-  }
-
-  #logo-container {
-   display: flex;
-   align-items: center;
-   text-align: center;
-   width: 100px;
-   margin-left: 24px;
-   /* margin-top: 14px; */
-  }
-
-  #navbar p {
-   color: var(--primary-color);
-   font-weight: 700;
-   font-size: 18px;
-   font-style: normal;
-   line-height: 20px;
-/*    width: 76px;
-   height: 20px;  */  
-  }
-
-  #menu {
-   /* display: flex;
-   flex-direction: row;
-   align-items: flex-start;
-   padding: 8px;
-   position: absolute;
-   width: 40px;
-   height: 40px;
-   right: 24px;
-   top: calc(50% - 40px/2); */
-   height: 40px;
-   width: 40px;
-   top: 4px;
-   border-radius: 8px;
-   padding: 8px;
-
-  }
-
-  #hamburger-container {
-   position: absolute;
-   flex: none;
-   width: 24px;
-   height: 24px;
-   display: flex;
-   flex-direction: row;
-   align-items: flex-start;
-   padding: 8px;
-   right: 24px;
+  /* NAVBAR */
+  #navbar-container {
+    display: flex;
+    justify-content: space-between;
+    height: 48px;
+    width: 100%;
+    margin-top: 44px;
```

Comment:
- [Optional] Good Job for strictly following the design 👍👍, I think in the Figma design the top part is showing the mobile default parts which are used to show the time, charger, and other notifications, so you don't need to add the space on your side also.
![Screenshot 2022-12-02 082135](https://user-images.githubusercontent.com/52010789/205221205-336cd853-b494-4469-9f51-1c48ea64ded6.png)